### PR TITLE
Feature/delete user

### PR DESCRIPTION
**Pull request by Semooze**
Implemented delete user with test cases.

### CODE REVIEW

File: tests/delete-user.fg [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# The login API is mounted in the odin view as a special
```
Comment (KayEss) [inline tests/delete-user.fg]: Let's not call this test file `delete-user.fg`. It expires the user, it doesn't delete the user. The name seems wrong

File: tests/CMakeLists.txt [hunk 2]
```diff
@@ -290,6 +286,29 @@ add_custom_command(OUTPUT test-login-google
             login-google.fg
     )
 
+add_custom_command(OUTPUT test-delete-user
```
Comment (KayEss) [inline tests/CMakeLists.txt]: Let's talk about expiry here rather than deletion.

File: Configuration/odin-views.json [hunk 3]
```diff
@@ -5,6 +5,7 @@
             "configuration": {
                 "": "odin/static",
                 "api/": "odin/api",
+                "api/delete/": "odin/delete",
```
Comment (KayEss) [inline Configuration/odin-views.json]: Why is this done here? Aren't there user APIs somewhere else?

Comment (Semooze) [reply Configuration/odin-views.json]: Just copy solution from similar other APIs. Yes it can be used in `api/`.

**Comment by KayEss** [comment]
Can you add the changelog and then merge it?

**Pull request closed without merging**